### ISSUE TITLE
Show buddies list in the main screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#79]
```
Add buddies list in the main screen with their connection status.
By selecting a buddy we can see if there is multiple connections.
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 6 Feb 2011 at 12:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 6 Feb 2011 at 12:45

Attachments:
- [screenshot_main_connected.png](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-79/comment-1/screenshot_main_connected.png)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9
```

Original comment by `Florent....@gmail.com` on 6 Feb 2011 at 8:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
